README.md

```diff
@@ -88,15 +88,15 @@ Add the plugin to your `build.gradle.kts`:
 
 ```kotlin
 plugins {
-  id("com.apollographql.apollo3") version "4.0.0-beta.7"
+  id("com.apollographql.apollo") version "4.0.0-beta.7"
```

Comment:
This reminds me we need to update `release.main.kts`

Reply:
This was done in https://github.com/apollographql/apollo-kotlin/commit/8d37c6119a43a620be175841050913c246ec1139 (but needs to be tested)